dpnp/backend/extensions/lapack/CMakeLists.txt

```diff
@@ -35,12 +35,10 @@ set(_module_src
     ${CMAKE_CURRENT_SOURCE_DIR}/getrf_batch.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/getri_batch.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/getrs.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/heevd.cpp
```

Comment:
What do you think to keep `heevd.cpp` and `syevd.cpp` files? (something similar to what have been done for `ufunc` extension, where we have `fabs.hpp`, `fabs.cpp` and source and header files for common functionality: `common.hpp`, `common.cpp`)